lkmpg.tex

```diff
@@ -1366,7 +1366,7 @@ \section{Replacing Print Macros}
 \subsection{Replacement}
 % FIXME: cross-reference
 In Section \ref{sec:using_x}, I said that X Window System and kernel module programming do not mix.
-That is true for developing kernel modules, but in actual use, you want to be able to send messages to whichever tty the command to load the module came from.
+That is true for developing kernel modules. But in actual use, you want to be able to send messages to whichever tty the command to load the module came from.
```

Comment:
Move the sentence staring from "But in actual .." to the next line. We use explicit newline to split each sentence.